### PR TITLE
effect vs label

### PR DESCRIPTION
**Pull request by dannytoomey**
Hi Witold,

Here's a new PR that resolves issue #150 without .so files or changes to Makevars. 

Best regards,

Danny Toomey

### CODE REVIEW

**Comment by dannytoomey** [comment]
Don't merge yet, this does not pass build checks.

File: R/baggr.R [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #'
 #' @param data data frame with summary or individual level data to meta-analyse;
 #'             see Details section for how to format your data
-#' @param model if \code{NULL}, detected automatically from input data
+#' @param label if \code{NULL}, detected automatically from input data
```
Comment (wwiecek) [inline R/baggr.R:14]: This is wrong. Did you mean to replace `effect` with `label`? `model` is something wrong

File: R/baggr.R [hunk 2]
```diff
@@ -224,6 +224,13 @@ baggr <- function(data,
   # if(!inherits(data, "data.frame") || nrow(data) == 1)
     # stop("data argument must be a data.frame of >1 rows")
 
+  # get label agrument and pass it as effect for compatibility
+  if("effect" %in% attributes(list(...))$names){
```
Comment (wwiecek) [inline R/baggr.R:228]: You need to write a few automated unit tests to prove that passing `effect` argument works the same way as `label`.

**Review by wwiecek — CHANGES_REQUESTED** [review]

File: R/baggr.R (hunk 2 above)
Comment (dannytoomey) [reply R/baggr.R:228]: Got it! There a few existing tests that used the `effect` argument, so I added tests to check that `label` and `effect` can be passed interchangeably there.

File: tests/testthat/test_binary.R [hunk 3]
```diff
@@ -60,11 +60,21 @@ bg5_rr <- expect_warning(
...
+bg5_rr_l <- expect_warning(
+  baggr(df_summ[,c("group", "a", "n1", "c", "n2")], label = "logRR",
+                iter = 150, chains = 2, refresh = 0,
+                show_messages = F))
+
+test_that("`effect` and `label` can be used the same way",{
+  expect_is(bg5_rr_l, "baggr")
+  expect_equal(bg5_rr_l$label, "logRR")
+})
+
```
Comment (dannytoomey) [inline tests/testthat/test_binary.R:72]: Added tests here

File: R/baggr.R (hunk 1 above)
Comment (dannytoomey) [reply R/baggr.R]: Yes, this was a mistake. Fixed this.

**Pull request closed without merging**